product-service/src/services/products/products.ts

```diff
@@ -0,0 +1,73 @@
+import { v4 } from "uuid";
+import { Product } from "src/shared/types/products";
+import { dbClient } from "../dynamo";
+
+export const generateProductId = () => v4();
+
+export const storeProduct = async (product: Omit<Product, "id">) => {
+  const { stocks, ...data } = product;
+  const itemId = generateProductId();
+
+  const item = {
+    ...data,
+    id: itemId,
+  }
+
+  try {
+    await dbClient.batchWrite({
+      RequestItems: {
+        [process.env.TABLE_PRODUCTS]: [
+          {
+            PutRequest: {
+              Item: item,
+            }
+          }
+        ],
+        [process.env.TABLE_STOCKS]: [
+          {
+            PutRequest: {
+              Item: {
+                product_id: itemId,
+                count: Number(stocks)
+              }
+            }
+          },
+        ]
+      }
+    }).promise();
+
+    return item;
+  } catch (err) {
+    console.log({ error: err });
+  }
+}
+
+// export const storeProductStock = async (itemData: ItemData) => {
```

Comment:
I think it would be better to remove commented code to make the codebase cleaner since this comment doesn't explain domain values